sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md

```diff
@@ -26,7 +26,7 @@ Configuration Reference:
 
 | Config Property Name      | Default | Description |
 | :---        |    :----   |         :--- | 
-| `spark.cosmos.write.strategy`      | `ItemOverwrite`    | Cosmos DB Item write Strategy: `ItemOverwrite` (using upsert), `ItemAppend` (using create, ignore pre-existing items i.e., Conflicts)  |
+| `spark.cosmos.write.strategy`      | `ItemOverwrite`    | Cosmos DB Item write Strategy: `ItemOverwrite` (using upsert), `ItemAppend` (using create, ignore pre-existing items i.e., Conflicts), `ItemDelete` (delete all documents), `ItemDeleteIfNotModified` (delete all documents for which the etag hasn't changed)  |
```

Comment:
did any customer ask for this feature?

Reply:
Yes - two customers both working on new workloads (and so using the new Spark connector) have asked for this feature or a sample in the notebooks on how to do bulk deletes directly via java sdk. But the latter would be much more complicatedfor customers to use.